xikolo-ios/VideoViewController.swift

```diff
@@ -15,24 +15,55 @@ class VideoViewController: UIViewController {
     @IBOutlet weak var containerVideoView: UIView!
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var descriptionTextView: UITextView!
+    @IBOutlet weak var openSlidesButton: UIButton!
+    @IBOutlet weak var previousItemButton: UIButton!
+    @IBOutlet weak var nextItemButton: UIButton!
+    @IBOutlet weak var containerHeighConstraint: NSLayoutConstraint!
     
-    var courseItem: CourseItem! {
+    var courseItem: CourseItem!
+    var video: Video? = nil {
         didSet {
-            updateUI()
+            videoDidLoad()
```

Comment:
Don't do this! I'm not sure if it's currently a problem, but do not call a VC's life cycle methods manually.